mgmt/smp/transport/smp_uart/src/smp_uart.c

```diff
@@ -354,42 +360,42 @@ nmgr_uart_rx_char(void *arg, uint8_t data)
         }
     }
     /* failed */
-    nus->nus_rx->omp_len = 0;
+    sus->sus_rx->omp_len = 0;
     m->om_len = 0;
     os_mbuf_free_chain(SLIST_NEXT(m, om_next));
     SLIST_NEXT(m, om_next) = NULL;
     return 0;
 }
 
 void
-nmgr_uart_pkg_init(void)
+smp_uart_pkg_init(void)
 {
-    struct nmgr_uart_state *nus = &nmgr_uart_state;
+    struct smp_uart_state *sus = &smp_uart_state;
     int rc;
     struct uart_conf uc = {
         .uc_speed = MYNEWT_VAL(NMGR_UART_SPEED),
         .uc_databits = 8,
         .uc_stopbits = 1,
         .uc_parity = UART_PARITY_NONE,
         .uc_flow_ctl = UART_FLOW_CTL_NONE,
-        .uc_tx_char = nmgr_uart_tx_char,
-        .uc_rx_char = nmgr_uart_rx_char,
-        .uc_cb_arg = nus
+        .uc_tx_char = smp_uart_tx_char,
+        .uc_rx_char = smp_uart_rx_char,
+        .uc_cb_arg = sus
     };
 
     /* Ensure this function only gets called by sysinit. */
     SYSINIT_ASSERT_ACTIVE();
 
-    rc = nmgr_transport_init(&nus->nus_transport, nmgr_uart_out, nmgr_uart_mtu);
+    rc = smp_transport_init(&sus->sus_transport, smp_uart_out, smp_uart_mtu);
     assert(rc == 0);
 
-    nus->nus_dev =
+    sus->sus_dev =
       (struct uart_dev *)os_dev_open(MYNEWT_VAL(NMGR_UART), 0, &uc);
```

Comment:
NMGR_UART name is change I guess to SMP_UART

Reply:
removed smp_uart